beacon-chain/rpc/service.go

```diff
@@ -212,6 +218,28 @@ func (s *Service) AttestHead(ctx context.Context, req *pb.AttestRequest) (*pb.At
 	return &pb.AttestResponse{AttestationHash: h[:]}, nil
 }
 
+// LatestAttestation streams the latest processed attestations to the rpc clients.
+func (s *Service) LatestAttestation(req *empty.Empty, stream pb.BeaconService_LatestAttestationServer) error {
+	sub := s.attestationService.IncomingAttestationFeed().Subscribe(s.incomingAttestation)
+	defer sub.Unsubscribe()
+
+	for {
+		select {
+		case attestation := <-s.incomingAttestation:
+			log.Info("Sending attestation to RPC clients")
+			if err := stream.Send(attestation); err != nil {
+				return err
+			}
+		case <-sub.Err():
+			log.Debug("Subscriber closed, exiting goroutine")
+			return nil
+		case <-s.ctx.Done():
+			log.Debug("RPC context closed, exiting goroutine")
+			return nil
```

Comment:
Please return a "server closed stream" error to the client. This branch is likely to occur when the server shuts down so it would be helpful for the client to understand that the stream was close unexpectedly.